### DISCUSSION TITLE
More community strings in one generator.yml

### DISCUSSION THREAD

**Discussion by tom-i** [prometheus/snmp_exporter#939] [Q&A] [answered]
I need to check more network vedors and I need to check more OIDs, so I would like to have one snmp.yml for snmp_exporter with more snmp v2 (more communities) and also v3 with authentifications.
So is it possible to add more "community" strings together in one generator.yml file?

Or is it better to have separate snmp.yml files for each vendor?

**Comment by R2NetD2** [comment]
Looks like we are all fighting the same issue since v0.23.0... 
https://github.com/prometheus/snmp_exporter/discussions/966

**Comment by tom-i** [comment, marked as answer]
I finally did it with 0.23:
https://discuss.prometheus.io/t/more-auths-in-0-23-snmp-exporter/1804/4

**Reply by R2NetD2** [reply]
Thanks!!! That's what I was trying so far, but failed on calling the auth from the module. Appreciate your help!
